x-pack/test/functional/services/ml/add_trained_models_flyout.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import type { FtrProviderContext } from '../../ftr_provider_context';
+
+export type AddTrainedModelUser = 'power' | 'viewer';
+
+export function TrainedModelsFlyoutProvider({ getService }: FtrProviderContext) {
+  const find = getService('find');
+  const testSubjects = getService('testSubjects');
+
+  const twentySeconds = 10000 * 2;
+
+  return new (class ModelsFlyout {
+    public async assertElserModelHeaderCopy(): Promise<void> {
+      const selector = 'div.euiFlyoutBody div p div.euiText';
```

Comment:
We should avoid such selectors, EUI can change the underlying structure. Could you please user test subject instead?

Reply:
I did start with that, but then it was continuously flaky, so I switched.  
Maybe I'm doing something wrong with where I'm encoding the data test subjects?